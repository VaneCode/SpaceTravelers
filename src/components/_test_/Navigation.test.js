import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import React from 'react';
import { Provider } from 'react-redux';
// import TestRenderer from "react-test-renderer";
import store from '../../redux/configureStore';
import Navigation from '../Navbar/Navbar';

describe('Navigation test', () => {
  test('snapshot', () => {
    const tree = render(
      <React.StrictMode>
        <Provider store={store}>
          <Router>
            <Navigation />
          </Router>
        </Provider>
      </React.StrictMode>,
    );
    expect(tree).toMatchSnapshot();
  });
});
