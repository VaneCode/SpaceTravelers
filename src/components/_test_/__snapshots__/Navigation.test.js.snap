// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation test snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar navbar-expand-lg navbar-light bg-light"
        style="border-bottom: 0.01rem solid #888; width: 100%; padding: 0.5rem;"
      >
        <div>
          <img
            alt="logo"
            src="planet.png"
            style="margin-right: 0.5rem; margin-left: 1rem;"
          />
        </div>
        <a
          class="mr-auto p-2 navbar-brand"
          href="/"
        >
          <a
            class="brand"
            href="/"
          >
            Space Traveler's Hub
          </a>
        </a>
        <button
          aria-controls="Navbar toggle"
          aria-label="Toggle navigation"
          class="navbar-toggler collapsed"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="justify-content-end navbar-collapse collapse"
          id="basic-navbar-nav"
        >
          <div
            class="navbar-nav"
          >
            <a
              class="nav-link"
              href="#"
              role="button"
              tabindex="0"
            >
              <a
                href="/"
                style="text-decoration: underline;"
              >
                Rockects
              </a>
            </a>
            <a
              class="nonActive nav-link"
              href="#"
              role="button"
              tabindex="0"
            >
              <a
                href="/missions"
                style="text-decoration: none;"
              >
                Missions
              </a>
            </a>
            <div
              class="line"
            />
            <a
              class="nonActive nav-link"
              href="#"
              role="button"
              tabindex="0"
            >
              <a
                href="/profile"
                style="text-decoration: none;"
              >
                My profile
              </a>
            </a>
          </div>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar navbar-expand-lg navbar-light bg-light"
      style="border-bottom: 0.01rem solid #888; width: 100%; padding: 0.5rem;"
    >
      <div>
        <img
          alt="logo"
          src="planet.png"
          style="margin-right: 0.5rem; margin-left: 1rem;"
        />
      </div>
      <a
        class="mr-auto p-2 navbar-brand"
        href="/"
      >
        <a
          class="brand"
          href="/"
        >
          Space Traveler's Hub
        </a>
      </a>
      <button
        aria-controls="Navbar toggle"
        aria-label="Toggle navigation"
        class="navbar-toggler collapsed"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="justify-content-end navbar-collapse collapse"
        id="basic-navbar-nav"
      >
        <div
          class="navbar-nav"
        >
          <a
            class="nav-link"
            href="#"
            role="button"
            tabindex="0"
          >
            <a
              href="/"
              style="text-decoration: underline;"
            >
              Rockects
            </a>
          </a>
          <a
            class="nonActive nav-link"
            href="#"
            role="button"
            tabindex="0"
          >
            <a
              href="/missions"
              style="text-decoration: none;"
            >
              Missions
            </a>
          </a>
          <div
            class="line"
          />
          <a
            class="nonActive nav-link"
            href="#"
            role="button"
            tabindex="0"
          >
            <a
              href="/profile"
              style="text-decoration: none;"
            >
              My profile
            </a>
          </a>
        </div>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
